src/Pages/Detail/Review.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import ReviewComment from "./ReviewComment";
+import "./Review.scss";
+
+class Review extends Component {
+  constructor() {
+    super();
+    this.state = {
+      reviewData: [
+        {
+          USER: {
+            id: "김댓글",
+            usercomment: "최고에요!",
+            date: "2020.09.03",
+            time: "22:09:26",
+          },
+
+          HOST: {
+            id: "호스트님의 답글",
+            hostcomment: "감동적인 후기 정말 감사합니다.",
+            date: "2020.09.04",
+            time: "23:08:25",
+          },
+        },
+        {
+          USER: {
+            id: "김댓글",
+            usercomment: "최고에요!",
+            date: "2020.09.03",
+            time: "22:09:26",
+          },
+
+          HOST: {
+            id: "호스트님의 답글",
+            hostcomment: "감동적인 후기 정말 감사합니다.",
+            date: "2020.09.04",
+            time: "23:08:25",
+          },
+        },
+      ],
+    };
+  }
+  render() {
+    return (
+      <div className="Review">
+        <div className="reviewHeaderBox">
+          <h4 className="reviewHeader">
+            이용 후기
+            <div className="reviewsNumber">
+              <span>1</span>개<span className="dot"></span>
+            </div>
+            <div className="average">
+              <span className="mark">평균 평점</span>
+              <span className="stars">0.0</span>
+            </div>
+          </h4>
+        </div>
+        {this.state.reviewData.map((el) => {
```

Comment:
굿굿!! map 활용 완전 많이 하셨네!!!
더 깔끔하게 보이기 위해서 비구조화도 한 번 사용해보세요!
```suggestion
   render(){
   const {reviewData} = this.state;
   return (
   ...
        {reviewData.map((el) => {
```

Reply:
넷